apps/pie-docs/test/system/cookie-banner.spec.js

```diff
@@ -0,0 +1,50 @@
+const COOKIE_NAMES = require('../../../../constants/cookies');
+
+describe('PIE - Cookie Banner Tests', async () => {
+    const cookieBannerSelector = '[data-test-id="cookie-banner-component"]';
+    const cookieAcceptAllSelector = '[data-test-id="accept-all-cookies-button"]';
+    const cookieNecessarySelector = '[data-test-id="accept-necessary-cookies-button"]';
+    const cookiesToDelete = Object.values(COOKIE_NAMES);
+
+    it('Should close the cookie banner when the first button is clicked', async () => {
+        const puppeteer = await browser.getPuppeteer();
+        const [page] = await puppeteer.pages();
+        const url = `${browser.options.baseUrl}`;
```

Comment:
Is this already a string?

Reply:
nice spot. yeah I think I can change this